services/feed/feed.go

```diff
@@ -42,6 +45,10 @@ func GetFeeds(ctx context.Context, opts activities_model.GetFeedsOptions) (activ
 func notifyWatchers(ctx context.Context, act *activities_model.Action, watchers []*repo_model.Watch, permCode, permIssue, permPR []bool) error {
 	// Add feed for actioner.
 	act.UserID = act.ActUserID
+	if len(act.Content) > maxActionContentLength {
```

Comment:
I do not think the `if` check is useful to end users or site admin.

If you think it is still useful to show the "warn", you could just use `EllipsisDisplayStringX` to check whether the text is truncated by checking `right!=""`